libraries/libraries_api.go

```diff
@@ -243,10 +281,10 @@ func (cll *ClusterLibraryList) String() string {
 
 // LibraryStatus is the status on a given cluster when using the libraries status api
 type LibraryStatus struct {
-	Library                         *Library `json:"library,omitempty"`
-	Status                          string   `json:"status,omitempty"`
-	IsLibraryInstalledOnAllClusters bool     `json:"is_library_for_all_clusters,omitempty"`
-	Messages                        []string `json:"messages,omitempty"`
+	Library  *Library `json:"library,omitempty"`
+	Status   string   `json:"status,omitempty"`
+	IsGlobal bool     `json:"is_library_for_all_clusters,omitempty"`
```

Comment:
should we start deprecation of this flag? Because per [documentation](https://docs.databricks.com/libraries/cluster-libraries.html#library):
>  This option does not install the library on clusters running Databricks Runtime 7.0 and above.

Reply:
It's not changeable by users anymore 🤷🏻‍♂️ but there still might be some workspaces with dbr7.0, that have this